.github/workflows/ci.yml

```diff
@@ -8,11 +8,16 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        toolchain: [stable]
```

Comment:
So you want stable here? And nightly for rustfmt and clippy? I’d have gone for nightly across the board.

Reply:
I'm not sure to understand the rationale of having the library being built against nightly. Wouldn't ensuring it compiles with the stable toolchain be preferable?
In general I prefer to stick to stable unless some unstable features are necessary.

Reply:
> In general I prefer to stick to stable unless some unstable features are necessary.

That's a fair position. I think the decision here is up to you and your team. I think it's fine to use nightly too, but I think I'm a minority and it's not a critical point.

You _may_ run into some trouble by having different rust versions to lint and format the code than what you're using in builds&tests. Nothing major and I for one prefer to avoid it.